tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -447,25 +447,20 @@ def test_field_activity_from_372__a():
     assert expected == result['field_activity']
 
 
-def test_name_variants_from_410__a_9():
+def test_name_variants_from_410__9_with_invalid_source():
```

Comment:
Shall we leave the previous test and add a new one that tests the case with invalid source?